src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$clock-height: 300px;
+$clock-width: 300px;
+$center-width: ($clock-width * 0.05);
+$center-height: ($clock-height * 0.05);
```

Comment:
`$clock-size: 300px; $center-width: $clock-width * 0.05;`, bc you want them to be equal

Reply:
Do you want me to create $clock-size var. instead of both $clock-width and $clock-height?
I cant do that because I need both of this values to make round shape of the clock (usind borger radius 50%)
or what do you mean?

Reply:
how clock-size variable affects on ability to make round shape using border radius?

Reply:
got it